npm-debug.log

```diff
@@ -0,0 +1,22 @@
+0 info it worked if it ends with ok
```

Comment:
Usuń ten plik, a następnie zrob commit, aby go permanentnie usunąć z repo.
(Jeśli już raz został dodany to nawet jak teraz dodasz do gitignore to już nic nie zmieni. Trzeba go usunąć i zrobić commit)